src/patternfly/layouts/Flex/flex.scss

```diff
@@ -21,6 +23,7 @@ $pf-l-flex--variable-map: build-variable-map("pf-l-flex--spacer", $pf-l-flex--sp
   display: var(--pf-l-flex--Display);
   flex-wrap: var(--pf-l-flex--FlexWrap);
   align-items: var(--pf-l-flex--AlignItems);
+  gap: var(--pf-l-flex--RowGap, --pf-l-flex--ColumnGap);
```

Comment:
did you mean to do it this way? 

```suggestion
  gap: var(--pf-l-flex--RowGap) var(--pf-l-flex--ColumnGap);
```

The way it's written now, `--pf-l-flex--ColumnGap` is a fallback for a single value being passed to `gap` (a single value will always evaluate as a `row-gap`), and since it isn't wrapped in `var()`, it's just an invalid value. You can see here that setting `--pf-l-flex--RowGap: initial`, which will force the fallback value to be used, results in no gap at all.

<img width="761" alt="Screenshot 2022-11-18 at 7 08 11 PM" src="https://user-images.githubusercontent.com/35148959/202826648-166d4e62-748f-43bd-9dc0-cce8d9a889ab.png">

Reply:
Yeah, I understand fallbacks; I just made a mistake.